importer/importer.py

```diff
@@ -166,6 +176,11 @@ def get_items(connection,
         class_to_use = SPECIFIC_TABLES[specific_table_name]["class"]
         data_files = load_data_files(
             SPECIFIC_TABLES[specific_table_name]["data_files"])
+        if SPECIFIC_TABLES[specific_table_name]["subclass_downloads"]:
```

Comment:
This should raise a `KeyError` if "subclass_downloads" is not present as a key.

Instead use `if specific_table.get("subclass_downloads")`